### DISCUSSION TITLE
Is there a way to control the scrollBehavior for the .scrollIntoView() or .focus() commands

### DISCUSSION THREAD

**Discussion by umar-ahmed** [ueberdosis/tiptap#2632] [General Questions]
The `scrollIntoView()` API for DOM supports as set of configuration options: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView, including alignment and ability to set `behavior: smooth` to get smooth scrolling. Is this something that's possible with the aforementioned commands?

**Comment by philippkuehn** [comment]
Unfortunately this isn’t supported by ProseMirror right now.
